.github/workflows/ecosystem_test.yaml

```diff
@@ -0,0 +1,56 @@
+name: Ecosystem test
+
+on:
+  workflow_call:
+    inputs:
+      repos_file:
+        description: 'Path to the file containing the list of repository names'
+        type: string
+        required: true
+        
+jobs:
+  update_and_test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: subosito/flutter-action@44ac965b96f18d999802d4b807e3256d5a3f9fa1
+        with:
+          channel: main
+
+      - run: echo "${{ toJSON(github.event.pull_request.labels.*.name) }}"
+
+      - run: dart pub global activate -s git https://github.com/mosuem/ecosystem.git --git-ref main --git-path pkgs/quest
```

Comment:
This should now reference `https://github.com/dart-lang/ecosystem`? Or, it should be updated after this PR lands?